<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addcontacto</class>
 <widget class="QMainWindow" name="addcontacto">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>200</width>
    <height>310</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>200</width>
    <height>310</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>200</width>
    <height>310</height>
   </size>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Agregar Contacto</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>D:/UPC/CICLO 3/Algoritmos y estructura de datos/Semana 7/logo_icon.ico</normaloff>D:/UPC/CICLO 3/Algoritmos y estructura de datos/Semana 7/logo_icon.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="sizeConstraint">
     <enum>QLayout::SetDefaultConstraint</enum>
    </property>
    <item>
     <widget class="QLabel" name="lblNom">
      <property name="styleSheet">
       <string notr="true">font: 12 pt &quot;Comic Sans MS&quot;;
color: rgb(128, 183, 102);</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Nombre:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="leNombre"/>
    </item>
    <item>
     <widget class="QLabel" name="lblApellido">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; color:#80b766;&quot;&gt;Apellido:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="leApellido"/>
    </item>
    <item>
     <widget class="QLabel" name="lblApod">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; color:#80b766;&quot;&gt;Apodo:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="leApodo"/>
    </item>
    <item>
     <widget class="QLabel" name="lblTel">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; color:#80b766;&quot;&gt;Telefono:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="leTelefono"/>
    </item>
    <item>
     <widget class="QPushButton" name="btnAC">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(52, 80, 113);
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Agregar Contacto</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnVolver">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(52, 80, 113);
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Volver</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>leNombre</tabstop>
  <tabstop>leApellido</tabstop>
  <tabstop>leApodo</tabstop>
  <tabstop>leTelefono</tabstop>
  <tabstop>btnAC</tabstop>
  <tabstop>btnVolver</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
